src/workerd/jsg/jsg.h

```diff
@@ -2296,6 +2296,8 @@ class Lock {
   void setCaptureThrowsAsRejections(bool capture);
   void setCommonJsExportDefault(bool exportDefault);
 
+  void setNodeJsCompatEnabled();
```

Comment:
A comment on what this enables (v1. vs. v2) would be nice here

Reply:
At this level it doesn't matter as jsg really has (and should not have) any information directly relating to compatibility flags. This is just a JSG specific configuration option that the compat flag happens to enable. We don't want any specific versioning on this.